test/vaadin-list-mixin.html

```diff
@@ -168,10 +170,11 @@
     }
 
     describe('vaadin-list-mixin', () => {
-      let list;
+      let list, list2;
```

Comment:
`list2` isn't very descriptive for a name. Rather `hiddenList`. If it makes sense, you could create a separate `describe('hidden-items', ...)` block for the tests with a list with hidden items.

Reply:
done